src/Hazelcast.Net.Tests/Serialization/Compact/CompactOptionsTests.cs

```diff
@@ -339,16 +339,6 @@ public void GetRegistrations()
 
             options.SetSchema(SchemaBuilder.For("duh").Build(), true);
             Assert.Throws<ConfigurationException>(() => options.GetRegistrations().ToList());
-
-            options = new CompactOptions();
```

Comment:
Why it's removed?

Reply:
It's just creating a new `CompactOptions` and not using it, probably a leftover, makes no sense so I removed it.

Reply:
Yeah but coming lines are also removed and the assertions were looking promising.

Reply:
weird - have put them back.